elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -130,8 +129,18 @@ $variables: (
   cursor: default !important;
 }
 
+// Default CTA fix for Firefox + arrow
+:host( :not([pfe-priority]):not([aria-disabled="true"]) ) {
+  .pfe-cta--wrapper {
```

Comment:
For consistency can we move this down below to the class-based styles since this is making an update to the wrapper class?

~line 153 would be a great spot to add:

```
@include browser-query(firefox) {
    :host( :not([pfe-priority]):not([aria-disabled="true"]) ) & {
        max-width: calc(100% - 8px - #{pfe-local($cssvar: size, $region: arrow)});
    }
}
```

Reply:
Great suggestion, thanks! updated